src/VisualStudio/Core/Def/DocumentOutline/DocumentSymbolUIItem.cs

```diff
@@ -32,12 +29,10 @@ public bool IsExpanded
         {
             get
             {
-                _threadingContext.ThrowIfNotOnUIThread();
                 return _isExpanded;
             }
             set
             {
-                _threadingContext.ThrowIfNotOnUIThread();
```

Comment:
if this doens't have a threading invariant, how do we prevent races when changing values?